unit_tests/test_redis_client_base.py

```diff
@@ -0,0 +1,176 @@
+import uuid
+from unittest import mock
+
+import pytest
+from redis import StrictRedis
+import d3a_api_client
+from d3a_api_client.redis_client_base import RedisClientBase, RedisAPIException
+from unittest.mock import MagicMock
+from unittest.mock import Mock
+from unittest.mock import patch
```

Comment:
Nit: these imports can be refactored to fit in one line:
```suggestion
from unittest.mock import MagicMock, Mock, patch
```
As a general comment: if you're using PyCharm, you can use the `Optimize Imports` feature to sort imports in the right order. You can find more information on our confluence wiki :)